### PR TITLE
Rain detect django

### LINKED ISSUE #31: [PROJECT] / [PROJECT IDEA]
I can add a an aim trainer for the intermidiate.

### PR DESCRIPTION
**Pull request by Virtual4087**
## Check List (Check all the applicable boxes) 

- [ x] I have updated the documentation accordingly.
- [x ] I have added a new Project.
- [x ] I have added a README.md file for the Project.
- [x ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by Virtual4087** [comment]
Hey man can you check it now? There's no secrets uncovered but it's still showing the same error

File: Projects/Django-Projects/rain_detect/README.md
Comment (sree-hari-s) [inline Projects/Django-Projects/rain_detect/README.md:1]: update the readme files on how to use the secrets to include your api keys.

**Comment by sree-hari-s** [comment]
> Hey man can you check it now? There's no secrets uncovered but it's still showing the same error

the issue is because from the previous commits we can access the secrets so that why its still showing the issue I think you might need to revert that commit so it is erased off of git history.

**Comment by Virtual4087** [comment]
Ohh ok I'll do that

File: Projects/Django-Projects/rain_detect/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Python == 3.11.3
+Django == 4.1
```
Comment (sree-hari-s) [inline Projects/Django-Projects/rain_detect/requirements.txt:2]: pyowm missing

**Comment by sree-hari-s** [comment]
this application seems to be just showing yes or no in a website, It would be good if you create a UI to input the place name instead of in the url.

**Comment by Virtual4087** [comment]
let me close this one and open another pr

**Pull request closed without merging**